src/vi.rs

```diff
@@ -9,29 +9,115 @@ use std::rc::Rc;
 
 #[derive(DeviceBE)]
 pub struct Vi {
+    // [1:0] type[1:0] (pixel size)
+    //     0: blank (no data, no sync)
+    //     1: reserved
+    //     2: 5/5/5/3 ("16" bit)
+    //     3: 8/8/8/8 (32 bit)
+    // [2] gamma_dither_enable (normally on, unless "special effect")
+    // [3] gamma_enable (normally on, unless MPEG/JPEG)
+    // [4] divot_enable (normally on if antialiased,
+    //     unless decal lines)
+    // [5] reserved - always off
+    // [6] serrate (always on if interlaced, off if not)
+    // [7] reserved - diagnostics only
+    // [9:8] anti-alias (aa) mode[1:0]
+    //     0: aa & resamp (always fetch extra lines)
+    //     1: aa & resamp (fetch extra lines if needed)
+    //     2: resamp only (treat as all fully covered)
+    //     3: neither (replicate pixels, no interpolate)
+    // [11] reserved - diagnostics only
+    // [15:12] reserved
```

Comment:
As a followup, if you check `StatusFlags` in `dp.rs`, I implemented a way to connect a `bitflags!` to a register. It still requires some glue (namely, `impl RegDeref` and a method like `Dp::cmd_status_ref`) which I was planning to generate through the `Device` macro, by adding something like this:

```rust
    #[reg(bank = 0, offset = 0xC, rwmask = 0, wcb, bitflags = StatusFlags)]
    cmd_status: Reg32,
```

I still need to work out the details, but this is something "rustish" to do in the future.

I think it might be worth using a bitflags for registers with many different bits like this one. For simpler registers maybe it's overkill, I don't mind a few manual bit accesses here and there.

Reply:
Totally agree, that would be nice. But I think for now we can add the `bitflags` versions of those registers, when we start actively using individual flags. 

On another note, there is https://github.com/phil-opp/rust-bit-field which is really nice for accessing individual bits.